frontend/src/scenes/notebooks/NotebookSelectButton/NotebookSelectButton.tsx

```diff
@@ -0,0 +1,235 @@
+import { LemonInput } from 'lib/lemon-ui/LemonInput/LemonInput'
+import { notebookLogicType } from '../Notebook/notebookLogicType'
+import { notebookNodeLogicType } from '../Nodes/notebookNodeLogicType'
+import { FlaggedFeature } from 'lib/components/FlaggedFeature'
+import { FEATURE_FLAGS } from 'lib/constants'
+import { ReactChild, useEffect } from 'react'
+import { LemonDivider } from '@posthog/lemon-ui'
+
+export type NotebookSelectProps = NotebookSelectButtonLogicProps & {
+    newNotebookTitle?: string
+    onNotebookOpened?: (
+        notebookLogic: BuiltLogic<notebookLogicType>,
+        nodeLogic?: BuiltLogic<notebookNodeLogicType>
+    ) => void
+}
+
+export type NotebookSelectPopoverProps = NotebookSelectProps &
+    Partial<PopoverProps> & {
+        children?: ReactChild
+    }
+
+export type NotebookSelectButtonProps = NotebookSelectProps &
+    Omit<LemonButtonProps, 'onClick' | 'children'> & {
+        newNotebookTitle?: string
+        onNotebookOpened?: (
+            notebookLogic: BuiltLogic<notebookLogicType>,
+            nodeLogic?: BuiltLogic<notebookNodeLogicType>
+        ) => void
+        onClick?: () => void
+        children?: ReactChild
+    }
+
+function NotebooksChoiceList(props: {
+    notebooks: NotebookListItemType[]
+    emptyState: string
+    onClick: (notebookShortId: NotebookListItemType['short_id']) => void
+}): JSX.Element {
+    return (
+        <div>
+            {props.notebooks.length === 0 ? (
+                <div className={'px-2 py-1'}>{props.emptyState}</div>
+            ) : (
+                props.notebooks.map((notebook, i) => {
+                    return (
+                        <LemonButton key={i} fullWidth onClick={() => props.onClick(notebook.short_id)}>
+                            {notebook.title || `Untitled (${notebook.short_id})`}
+                        </LemonButton>
+                    )
+                })
+            )}
+        </div>
+    )
+}
+
+export function NotebookSelectList(props: NotebookSelectProps): JSX.Element {
+    const logic = notebookSelectButtonLogic({ ...props })
+
+    const { resource, newNotebookTitle } = props
+    const { notebooksLoading, containingNotebooks, allNotebooks, searchQuery } = useValues(logic)
+    const { setShowPopover, setSearchQuery, loadContainingNotebooks, loadAllNotebooks } = useActions(logic)
+    const { createNotebook } = useActions(notebooksModel)
+
+    const openAndAddToNotebook = async (notebookShortId: string, exists: boolean): Promise<void> => {
+        await openNotebook(notebookShortId, NotebookTarget.Popover, null, (theNotebookLogic) => {
+            if (!exists && props.resource) {
+                theNotebookLogic.actions.insertAfterLastNode([props.resource])
+            }
+            props.onNotebookOpened?.(theNotebookLogic)
+        })
+    }
+
+    const openNewNotebook = (): void => {
+        const title = newNotebookTitle ?? `Notes ${dayjs().format('DD/MM')}`
```

Comment:
It looks like the only other place this is used is in the PlayerMetaLinks and is the same value: https://github.com/PostHog/posthog/blob/master/frontend/src/scenes/session-recordings/player/PlayerMetaLinks.tsx#L63. Maybe we can remove it from there in future?